tests/raw_query/models.py

```diff
@@ -35,5 +35,9 @@ class Reviewer(models.Model):
     reviewed = models.ManyToManyField(Book)
 
 
+class Product(models.Model):
+    price = models.DecimalField(max_digits=10, decimal_places=2)
+
```

Comment:
Please move this field to an existing model (`Coffee` should do) in order to prevent the creation of a table for a single test